tests/js/download-api-helper.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const username = process.argv[2];
+const realmName = process.argv[3];
+const realmModule = process.argv[4];
+
+var Realm = require(realmModule);
+Realm.Sync.User.register('http://localhost:9080', username, 'password', (error, user) => {
+    if (error) {
+        console.log(error);
+        process.exit(-2);
+    } else {
+        const config = {
+            sync: { user, url: `realm://localhost:9080/~/${realmName}`, error: err => console.log(err) },
+            schema: [{ name: 'Dog', properties: { name: 'string' } }]
+        };
+
+        var realm = new Realm(config);
+
+        realm.write(() => {
+            for (let i = 1; i <= 3; i++) {
+                realm.create('Dog', { name: `Lassy ${i}` });
+            }
+        });
+
+        console.log("Dogs count " + realm.objects('Dog').length);
+        setTimeout(_ => process.exit(0), 3000);

```

Comment:
The wait is there to allow the syncing to happen, ie not to end the process before the sync is done. Otherwise the test will fail always since we want to test the download api waited for the exact number of records.